draft-ietf-quic-transport.md

```diff
@@ -73,9 +73,11 @@ informative:
     title: "Structured Streams: A New Transport Abstraction"
     author:
       - ins: B. Ford
-    date: 2007-08
+    date: 2007-10
```

Comment:
over here: http://www.sigcomm.org/node/2648 - I guess August was the conference date, and October was the publication date.